### PR TITLE
Update Ledger doc with Keplr

### LINKED ISSUE #1202: docs: outdated link to Lunie
https://hub.cosmos.network/main/resources/ledger.html
last sentence First prgh

## Steps to Reproduce

____

#### For Admin Use

- [ ] Not duplicate issue
- [ ] Appropriate labels applied
- [ ] Appropriate contributors tagged
- [ ] Contributor assigned/self-assigned

### PR DESCRIPTION
**Pull request by nooomski**
Replaced Lunie instructions with Keplr. Closes #1202

### CODE REVIEW

File: docs/resources/ledger.md [hunk 1]
```diff
@@ -31,21 +31,18 @@ To see the `Cosmos` application when you search for it, you might need to activa
 
 ![Devmode](../images/ledger-tuto-dev-mode.png)
 
-## Lunie.io + Ledger Nano
```
Comment (glnro) [inline docs/resources/ledger.md:34]: I believe there's still a reference to Lunie at the top of the docs.

**Review by glnro — COMMENTED** [review]
Looks good otherwise, I might try to also include https://support.ledger.com/hc/en-us/articles/4411149814417-Set-up-and-use-Keplr-to-access-your-Ledger-Cosmos-ATOM-account?docs=true as well as https://support.ledger.com/hc/en-us/articles/360013713840-Cosmos-ATOM-?docs=true which explains their recommended workaround for not having done the work in preparation of the Vega upgrade...not sure if it's still relevant though.

File: docs/resources/ledger.md [hunk 2]
```diff
@@ -208,5 +205,5 @@ Feel welcome to reach out in our [Telegram channel](https://t.me/cosmosproject)
 
 Here are a few relevant and helpful tutorials from the wonderful Cosmos community:
 
-- [Ztake](https://medium.com/@miranugumanova) - [How to Redelegate Cosmos Atoms with the Lunie Web Wallet](https://medium.com/@miranugumanova/how-to-re-delegate-cosmos-atoms-with-lunie-web-wallet-8303752832c5)
+- [Catdotfish](https://catdotfish.medium.com/) - [How to use the Keplr Wallet](hhttps://medium.com/chainapsis/how-to-use-keplr-wallet-40afc80907f6)
```
Comment (zramsay) [inline docs/resources/ledger.md]: double H in the link (hhttps)

Comment (nooomski) [reply docs/resources/ledger.md]: Nice catch!

File: docs/resources/ledger.md (hunk 1 above)
Comment (nooomski) [reply docs/resources/ledger.md:34]: Whoopsie, quite a bit of them haha. Fixed now!

**Comment by nooomski** [comment]
Thanks @glnro !

> Looks good otherwise, I might try to also include https://support.ledger.com/hc/en-us/articles/4411149814417-Set-up-and-use-Keplr-to-access-your-Ledger-Cosmos-ATOM-account?docs=true as well as https://support.ledger.com/hc/en-us/articles/360013713840-Cosmos-ATOM-?docs=true which explains their recommended workaround for not having done the work in preparation of the Vega upgrade...not sure if it's still relevant though.

Removed additional Lunie references & added one of the links if users want more details. Don't think the vega upgrade issue is relevant anymore.

**Review by glnro — APPROVED** [review]

**Review by okwme — APPROVED** [review]

**Pull request merged**